app/src/main/java/org/fossasia/openevent/fragments/SpeakerFragment.java

```diff
@@ -33,6 +36,20 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         speakersListAdapter = new SpeakersListAdapter(dbSingleton.getSpeakerList());
         speakersRecyclerView.setAdapter(speakersListAdapter);
         speakersRecyclerView.setLayoutManager(new LinearLayoutManager(getActivity()));
+        speakersRecyclerView.addOnItemTouchListener(
+                new RecyclerItemClickListener(view.getContext(),
+                        new RecyclerItemClickListener.OnItemClickListener() {
+                            @Override
+                            public void onItemClick(View view, int position) {
+//                                String speaker_name = ((TextView) view.findViewById(R.id.speaker_name)).getText().toString();
+//                                Intent intent = new Intent(view.getContext(), SpeakersActivity.class);
+//                                intent.putExtra(SPEAKER,speaker_name);
+//                                startActivity(intent);
+                                if (Build.VERSION.SDK_INT > Build.VERSION_CODES.LOLLIPOP) {
+                                    view.setTranslationZ(16.0f);
```

Comment:
A simpler technique is 

```java
    public static int convertDip2Px(float nDip) {
        return (int) (SCREEN_DENSITY * nDip + 0.5f);
    }
```
where `SCREEN_DENSITY` can be obtained by `getResources().getDisplayMetrics().density`

Sizing the pixels and converting to int rounds it up (ceil) to the nearest integer.